docs/release-notes/v0.35.0.md

```diff
@@ -0,0 +1,78 @@
+## Announcing Radius v0.35.0
+
+Today we're happy to announce the release of Radius v0.35.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+<!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->
```

Comment:
### Azure Workload Identity support
//TODO @Reshrahim 

### Nested modules in Terraform Recipes
With this release, Radius now supports authentication into private repositories for pulling Terraform Recipes that contain nested modules. See the [private Terraform modules guide](https://docs.radapp.io/guides/recipes/terraform/howto-private-registry/) for more details.

### EnableWebsockets property added to the Gateway Route Resource
The `EnableWebsockets` property has been added to the Gateway Route Resource so that it can be set by Radius users to enable features that make use of websockets, such as SignalR. See the [Gateway resource schema](https://docs.radapp.io/reference/resource-schema/core-schema/gateway/#routes) for more details.